pylintrc

```diff
@@ -1,5 +1,5 @@
 [MASTER]
-load-plugins=pylint_mccabe,pylint_django
+load-plugins=pylint.extensions.mccabe,pylint_django
```

Comment:
We should synchronise this with other projects where McCabe is provided by pyflakes.